client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-view/motion-detail-view.component.ts

```diff
@@ -302,16 +321,51 @@ export class MotionDetailViewComponent extends BaseMeetingComponent implements O
         const indexOfCurrent = this._sortedMotions.findIndex(motion => motion === this.motion);
         if (indexOfCurrent > 0) {
             this.previousMotion = this._sortedMotions[indexOfCurrent - 1];
+            this.substituteSurroundingMotions(indexOfCurrent, true);
         } else {
             this.previousMotion = null;
         }
         if (indexOfCurrent > -1 && indexOfCurrent < this._sortedMotions.length - 1) {
             this.nextMotion = this._sortedMotions[indexOfCurrent + 1];
+            this.substituteSurroundingMotions(indexOfCurrent, false);
         } else {
             this.nextMotion = null;
         }
     }
 
+    /**
+     * Substitutes the previous or next motion.
+     * Fires if @var this._amendmentsInMainList as well as @var this._navigatedFromAmendmentList collide.
+     * @param indexOfCurrent The index from the active motion.
+     * @param previous Wether the method is called for the previous or the next motion.
+     */
+    private substituteSurroundingMotions(indexOfCurrent: number, previous: boolean) {
+        if (this._amendmentsInMainList && this._navigatedFromAmendmentList) {
+            let index = indexOfCurrent;
+            if (!previous) {
+                while (index < this._sortedMotions.length - 1) {
```

Comment:
I'd prefer a for loop here. When changing this to return the next suitable motion it should also be possible to reduce this into one loop.